tests/__init__.py

```diff
@@ -49,23 +49,30 @@ def udev_rules(self):
             self.reload_udev_rules()
 
     @pytest.fixture()
-    def events(self, device, endpoint=0):
-        events = []
+    def event_nodes_fd(self, device, endpoint=0):
+        fds = []
         for d in pyudev.Context().list_devices(subsystem='input'):
```

Comment:
We should probably change that statement in a future PR to get the list of event nodes from the endpoints themselves, not having to go through pyudev on all devices.

Reply:
Yes, that should be handled in #39.